02/Elena Davydik/src/components/Main/HotelsSearchForm/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { ChangeEvent } from "react";
+
+type Props = {
+    onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+}
+
+const searchFormTitle: string = 'Available hotels';
+
+export const HotelsSearchForm = ({onChange}: Props) => (
+    <div className="container">
+        <h2 className="title">{searchFormTitle}</h2>
+        <input onChange = {onChange} placeholder = 'Enter destination or hotel name' className="searchForm__input"/>
```

Comment:
you declared a variable for string on the line 7, but use hardcoded string for placeholder. move it in the variable as well

Reply:
Create the variable _inputPlaceholderText_ for input placeholder